### DISCUSSION TITLE
Undefined index: id ErrorException in PerformActionMiddleware

### DISCUSSION THREAD

**Discussion by goestijn** [livewire/livewire#3775] [7. Bugs (archived)] [answered]
After updating to the lasted version of livewire 2.6.3 I get an ErrorException on each action I execute saying 'Undefined index: id'.
Further investigation of this error showed this is thrown in the HydrationMiddleware\PerformActionCalls class.

Temporaliy downgrading back to version 2.5.5 seems to solve the issue.
Any sugestions how this error can be fixed. 

![image](https://user-images.githubusercontent.com/28931777/133639770-27481348-941e-428e-bb47-fc71adb53530.png)

**Comment by joshhanley** [comment, marked as answer]
@goestijn please see this pinned discussion for how to fix this https://github.com/livewire/livewire/discussions/3769

Hope this helps!
